### PR TITLE
Implement classroom creating feature

### PR DESCRIPTION
**Pull request by tonkaew131**
#17 

- Allow anyone to create a classroom (If "Free" package, check if create more than 3)
- Provide information to classroom
   - Name, Description, Thumbnail, Schedule
- Show lists of joined student

v Demo Video
[Demo video](https://cloud.wilar.in.th/s/Fqq6X65XPdFWf3j)

### CODE REVIEW

**Review by Gapraomoosap — APPROVED** [review]
I don't know what is this but It look so cool so approve

File: backend/lib/db.ts [hunk 1]
```diff
@@ -1,3 +1,90 @@
...
 
-export const sql = postgres(process.env.DATABASE_URL);
+import { fileExtension, uuidv4 } from './utils';
+
+export const sql = postgres(process.env.DATABASE_URL || '');
+
+export async function uploadFile(
+    file: File,
+    uploadById: string,
+    options: {
```
Comment (MonitorIizard) [inline backend/lib/db.ts:11]: what is public true? like anyone can access the file if `options: { public : true }` ?

**Review by MonitorIizard — APPROVED** [review]
good

File: learn-arai/components/hooks/useLocalStorage.ts
[the remark this answers is not in the document]
Comment (tonkaew131) [reply learn-arai/components/hooks/useLocalStorage.ts:1]: because hooks are reusable "component"

File: backend/lib/db.ts (hunk 1 above)
Comment (tonkaew131) [reply backend/lib/db.ts:11]: Default as public, yes. For false and other value, I'm going to implement access control later when needed

File: backend/schema/database-schema.sql
[the remark this answers is not in the document]
Comment (tonkaew131) [reply backend/schema/database-schema.sql:1]: I gotya fam

Updated Class Diagram

**Pull request merged**